subsys/net/openthread/Kconfig

```diff
@@ -26,7 +26,7 @@ config OPENTHREAD_NRF_SECURITY
 	  OpenThread to make use of hardware accelerated cryptography functions
 	  if available as well as fast oberon backend for software encryption.
 
-config OPENTHREAD_MBEDTLS_TARGET
+config OPENTHREAD_MBEDTLS_LIB_NAME
 	default "mbedtls_common" if OPENTHREAD_NRF_SECURITY
```

Comment:
Does it still make sense to have a sole "mbedtls_common" as default?

Reply:
With this pull I was planning only to pull in the upstream change. Another PR will be created this week that updates this value and some other options allowing use of cc310, oberon and vanilla mbedtls depending on the board used